ActindoWorkFlow/UserInterface/08_CoreActions.md

```diff
@@ -0,0 +1,247 @@
+# Workflow core actions
+
+![Core actions](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/CoreActions.png "[Core actions]")
+
+To create a workflow, you must include one or several workflow actions.   
+In the *Workflows* module, a certain number of workflow actions is always preconfigured, the so-called *Core actions*. These core actions are used to build control flow structures in a workflow.   
+Other workflow actions are available depending on the plugins installed in the current system.
```

Comment:
Ich würde alle drei Sätze oben zusamen in einem Abschnitt als Einführung. Nächster Satz dann separat (eine Zeile dazwischen).